external-crates/move/crates/move-binary-format/src/unit_tests/compatibility_tests.rs

```diff
@@ -1199,3 +1200,57 @@ fn check_new_changed_missing_declarations() {
     assert!(InclusionCheck::Subset.check(&empty, &m1).is_ok());
     assert!(InclusionCheck::Equal.check(&empty, &m1).is_err());
 }
+
+fn arbitrary_map() -> impl Strategy<Value = BTreeMap<i32, i32>> {
+    prop::collection::btree_map(0..1000, 0..1000, 0..20)
+}
+
+proptest! {
+    #[test]
+    fn test_compare_ord_iters(old in arbitrary_map(), new in arbitrary_map()) {
+        let result: Vec<_> = compare_ord_iters(old.iter(), new.iter()).collect();
+
+        for mark in &result {
+                match mark {
+                    Mark::New(key, _) => {
+                        // New keys must only be in `new`
+                        assert!(new.contains_key(key));
```

Comment:
I'd use [`prop_assert`]( https://altsysrq.github.io/rustdoc/proptest/latest/proptest/macro.prop_assert.html) and `prop_assert_eq` here and elsewhere in this function as it will help proptest narrow counterexamples if errors are found.